mod_cluster/extension/src/main/java/org/wildfly/extension/mod_cluster/ModClusterConfigResourceDefinition.java

```diff
@@ -299,6 +299,12 @@
     public static void buildTransformation(ModelVersion version, ResourceTransformationDescriptionBuilder parent) {
         ResourceTransformationDescriptionBuilder builder = parent.addChildResource(PATH);
 
+        if (ModClusterModel.VERSION_4_0_0.requiresTransformation(version)) {
+            builder.getAttributeBuilder()
+                    .setValueConverter(new AttributeConverter.DefaultValueAttributeConverter(EXCLUDED_CONTEXTS), EXCLUDED_CONTEXTS)
```

Comment:
This is funny, this needs a custom-made transformer... since the legacy won't take null for an answer :-)